examples/README.md

```diff
@@ -1,53 +0,0 @@
-Sample OAuth 2.0 servers and clients
```

Comment:
Did we construct the "where the bodies are buried" JIRA in order that we can find this material again? I couldn't find it by searching.